lib/core/src/Cardano/Wallet.hs

```diff
@@ -2049,7 +2054,7 @@ joinStakePool
     -> PoolId
     -> PoolLifeCycleStatus
     -> WalletId
-    -> ExceptT ErrJoinStakePool IO DelegationAction
+    -> ExceptT ErrJoinStakePool IO (DelegationAction, Coin)
```

Comment:
Maybe a bit inelegant, but don't see a way to stuff it into DelegationAction and creating a separate type also seems excessive.